src/mixins/itext_behavior.mixin.ts

```diff
@@ -92,11 +88,9 @@ export abstract class ITextBehaviorMixin<
    * Initializes all the interactive behavior of IText
    */
   initBehavior() {
-    this.initAddedHandler();
-    this.initRemovedHandler();
     this.initCursorSelectionHandlers();
     this.initDoubleClickSimulation();
-    this.mouseMoveHandler = this.mouseMoveHandler.bind(this);
+    this.updateSelection = this.updateSelection.bind(this);
```

Comment:
this method needs to be called something that is both mouse move and handler, so that we avoid it using differently later.
Like updateSelectionOnMouseMove or something like that.

Reply:
a237a9ac0